### PR TITLE
FRU updates

### PR DESCRIPTION
**Pull request by OTCompa**
Uncommenting out stuff and adding prerequisite things in preparation for FRU.

### CODE REVIEW

File: internal/clearingway/clears-handler.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func (c *Clearingway) ClearsHelper(s *discordgo.Session, i *discordgo.Interactio
 	if !isOwner {
 		err = discord.ContinueInteraction(s, i.Interaction,
 			fmt.Sprintf(
-				"I could not verify your ownership of `%s (%s)`!\nIf this is your character, add the following code to your Lodestone profile and then run `/clears` again:\n\n**%s**\n\nYou can edit your Lodestone profile at https://na.finalfantasyxiv.com/lodestone/my/setting/profile/",
+				"I could not verify your ownership of `%s (%s)`!\nIf this is your character, add the following code to your Lodestone profile and then try again:\n\n**%s**\n\nYou can edit your Lodestone profile at https://na.finalfantasyxiv.com/lodestone/my/setting/profile/",
```
Comment (brtran4) [inline internal/clearingway/clears-handler.go]:
```suggestion
				"I could not verify your ownership of `%s (%s)`!\nIf this is your character, add the following code to your Lodestone profile and try again:\n\n**%s**\n\nYou can edit your Lodestone profile at https://na.finalfantasyxiv.com/lodestone/my/setting/profile/",
```

File: internal/clearingway/legend-roles.go [hunk 2]
```diff
@@ -124,7 +124,21 @@ func LegendRoles() *Roles {
 					return true, output
 				}
 
-				return false, "Did not clear all five ultimates."
+				return false, "Did not clear only five ultimates."
```
Comment (brtran4) [inline internal/clearingway/legend-roles.go:127]: i think we should revert this string

Comment (OTCompa) [reply internal/clearingway/legend-roles.go:127]: Resolved from discord convo

File: internal/clearingway/legend-roles.go [hunk 3]
```diff
@@ -124,7 +124,21 @@ func LegendRoles() *Roles {
...
+			},
+		},
+		// TODO: change role name to what atmus wants
```
Comment (brtran4) [inline internal/clearingway/legend-roles.go]: since this is final let's get rid of this TODO

**Review by brtran4 — APPROVED** [review]

**Pull request merged**